packages/tokens/src/font.json

```diff
@@ -13,32 +13,29 @@
 	},
 	"fontSize": {
 		"$type": "dimension",
-		"0": {
-			"$value": "{size.10}"
-		},
 		"100": {
-			"$value": "{size.12}"
+			"$value": "{size.11}"
 		},
 		"200": {
-			"$value": "{size.14}"
+			"$value": "{size.13}"
 		},
 		"300": {
-			"$value": "{size.16}"
+			"$value": "{size.15}"
 		},
 		"400": {
-			"$value": "{size.18}"
+			"$value": "{size.20}"
 		},
 		"500": {
-			"$value": "{size.20}"
+			"$value": "{size.24}"
 		},
 		"600": {
-			"$value": "{size.24}"
+			"$value": "{size.32}"
 		},
 		"700": {
-			"$value": "{size.32}"
+			"$value": "{size.44}"
 		},
 		"800": {
-			"$value": "{size.44}"
+			"$value": "3.75rem"
```

Comment:
is there a reason this doesn't have a corresponding `size` token?

Reply:
Tentative value so we wanted to wait before adding it as a size.